<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SearchDialog</class>
 <widget class="QDialog" name="SearchDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>779</width>
    <height>471</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="searchButton">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>20</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Search</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>410</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Input keyword to search:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="keyWord">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>20</y>
     <width>311</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableView" name="tView">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>621</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextBrowser" name="tBrowser">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>260</y>
     <width>311</width>
     <height>192</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>AR PL UMing CN</family>
    </font>
   </property>
  </widget>
  <widget class="QTextBrowser" name="tBrowser_2">
   <property name="geometry">
    <rect>
     <x>345</x>
     <y>260</y>
     <width>301</width>
     <height>192</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>AR PL UMing CN</family>
    </font>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>SearchDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>710</x>
     <y>416</y>
    </hint>
    <hint type="destinationlabel">
     <x>710</x>
     <y>395</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
